app.py

```diff
@@ -427,6 +432,25 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
     streamlit.plotly_chart(figure_or_data=collateral_figure, use_container_width=True)
     streamlit.plotly_chart(figure_or_data=debt_figure, use_container_width=True)
 
+    class ProtocolColors:
```

Comment:
why do you have it inside the main function, we are trying to move out as much as possible from here, to make it more readable and maintainable. Please move it out, You have `src/chart_utils.py` for it, move it there please,